lib/twitter-ads/http/ton_upload.rb

```diff
@@ -39,6 +39,7 @@ def initialize(client, file_path, opts = {})
 
       @client    = client
       @bucket    = opts.delete(:bucket) || DEFAULT_BUCKET
+      @min_file_size = opts.delete(:min_file_size) || MIN_FILE_SIZE
```

Comment:
`MIN_FILE_SIZE` is actually a hard limit enforced in the TON API and it's not meant to be configurable. It's a limit we've seen most API consumers be successful with. Larger files should be chunked for best performance.